pkg/crc/machine/start.go

```diff
@@ -366,10 +366,6 @@ func (client *client) Start(ctx context.Context, startConfig types.StartConfig)
 		logging.Warn(fmt.Sprintf("Failed to query DNS from host: %v", err))
 	}
 
-	if err := cluster.EnsurePullSecretPresentOnInstanceDisk(sshRunner, startConfig.PullSecret); err != nil {
```

Comment:
Doesn that mean the MCO will restart Kubelet to reload the pull secret ? Or the pull secret is reloading without restart Kubelet ?

Reply:
I also recall that some containers won't start if this file is not present. This line removed a lot of crc start failures in fact.